lelkek.FrogBox/team.php

```diff
@@ -0,0 +1,109 @@
+ ___/\\\/////////\\\___________\/\\\_______\/\\\_______________/\\\////////________________/\\\////////__       
+  __\//\\\______\///____________\/\\\_______\/\\\_____________/\\\/_______________________/\\\/___________      
+   ___\////\\\___________________\/\\\_______\/\\\____________/\\\________________________/\\\_____________     
+    ______\////\\\________________\/\\\_______\/\\\___________\/\\\_______________________\/\\\_____________    
+     _________\////\\\_____________\/\\\_______\/\\\___________\//\\\______________________\//\\\____________   
+      __/\\\______\//\\\____________\//\\\______/\\\_____________\///\\\_____________________\///\\\__________  
+       _\///\\\\\\\\\\\/______________\///\\\\\\\\\/________________\////\\\\\\\\\______________\////\\\\\\\\\_ 
+        ___\///////////__________________\/////////_____________________\/////////__________________\/////////__
+-->
+  </head>
+
+  <body>
+    <div class="container">
+
+      <!-- START NAVBAR -->
+
+      <div class="navbar">
+        <a href="https://frogboxentertainment.com"><img id="logo" src="./fbl.png" alt="FrogBox Logo" title="Frouje Bouckes"></a>
+        <a id="logotext" href="https://frogboxentertainment.com">FROGBOX</a>
+        <ul id="succ">
+
+          <!-- COPY PASTE AN "<a href="home.html"><li class="divided">Home</li></a>" TO CREATE A NEW TAB IN THE NAVIGATION BAR ||| MAKE SURE TO CHANGE THE HREF TO THE PATH YOU WANT -->
+
+          <a href="./index">
+            <li class="divided">Home</li>
+          </a>
+          <a href="./games">
+            <li class="divided">Games</li>
+          </a>
+          <a href="./team">
+            <li class="divided" style="background: #b5d849; color: white;">Team</li>
+          </a>
+          <a href="./store">
+            <li class="divided">Store</li>
+          </a>
+          <a href="./contact">
+            <li class="divided">Contact</li>
+          </a>
+        </ul>
+      </div>
+
+      <!-- END NAVBAR -->
+
+      <!-- START TEAM MEMBER DISPLAY SHEET - MORE DOCUMENTATION LISTED BELOW -->
+
+      <div class="teamtitle">
+        Our Team
+      </div>
+
+      <div class="teamwrapper">
+
+<?php
+$servername = "localhost";
+$username = "frogbox";
+$password = "noooooooooooooooooooope";
+$dbname = "frogbox";
+
+// connect af
+$conn = new mysqli($servername, $username, $password, $dbname);
+// for debugging(kinda)
+if ($conn->connect_error) {
+    die("Connection failed: " . $conn->connect_error);
+}
+
+$sql = "SELECT image, name, description FROM employees";
+$result = $conn->query($sql);
+
+if ($result->num_rows > 0) {
+    // output data of each row
+    while($row = $result->fetch_assoc()) {
+        echo "<div class=\"arrowhover teammember\"><img  class=\"teamimg\" src=\"" . $row['image'] . "\"><p class=\"nametext\">" . $row['name'] . "</p><div class=\"arrow_box regular\">" . $row['description'] . "</div></div>";
```

Comment:
Oh so there's the DRY components. So why do you have a php & non php version of pages? If only you had explained this in your readme...